docs/volt.md

```diff
@@ -682,3 +682,20 @@ If your Volt component is nested, you may use "dot" notation to specify the comp
 ```php
 Volt::test('users.stats')
 ```
+
+#### Testing a page contains a Volt component
+
+The simplest way to test if a given endpoint contains a Volt component is by providing `assertSeeLivewire` with the 
+encoded component alias using the `FragmentAlias::encode` method. 
+
+Where the first argument is the name passed into the `@volt` Blade directive, and the second is the path to where 
+the Volt component is located at:
+
+```php
+use Livewire\Volt\FragmentAlias;
+
+$this->get('/posts/create')
+    ->assertSeeLivewire(
+        FragmentAlias::encode('posts.create', resource_path('views/livewire/posts/create.blade.php'))
+    );
```

Comment:
Or would it be better to suggest using `FragmentMap::get`? That also works but the tests do run a bit slower due to the look-up.

```suggestion
use Livewire\Volt\FragmentMap;

$this->get('/posts/create')
    ->assertSeeLivewire(
        FragmentMap::get('posts.create')
    );
```

Reply:
I've decided against suggesting this, upon further testing it sometimes doesn't find the component, could possibly be because a component isn't in the look-up. I don't know the inner workings of Volt so I'm basing this on a rudimentary look of the source.